cli/JSXV4UPGRADE.md

```diff
@@ -0,0 +1,43 @@
+**JSX V4 upgrade path**
+
+1. No need to upgrade rescript-react or change bsconfig.json
+
+   JSX V3 by default.
+
+2. New opt-in configuration added in bsconfig.json
+
+   a. JSX V3
+
+   ```json
+   "jsx": {
+     "version": 3,
+     "module": "react",
+     "mode": "classic"
+   }
+   ```
+
+   **Note:** When using `jsx` v3, the `runtime` option will be ignored.
```

Comment:
this line seems old

Reply:
Wow, it looks old now. Docs are easy to be outdated.